bin/pytorch_inference/Main.cc

```diff
@@ -170,6 +171,18 @@ void writeThreadSettings(ml::core::CJsonOutputStreamWrapper& wrappedOutputStream
     jsonWriter.EndObject();
 }
 
+void writeSimpleAck(ml::core::CJsonOutputStreamWrapper& wrappedOutputStream,
+                    const std::string& requestId) {
+    ml::core::CRapidJsonConcurrentLineWriter jsonWriter(wrappedOutputStream);
+    jsonWriter.StartObject();
+    jsonWriter.Key(ACK);
+    jsonWriter.StartObject();
+    jsonWriter.Key(ml::torch::CCommandParser::REQUEST_ID);
+    jsonWriter.String(requestId);
+    jsonWriter.EndObject();
+    jsonWriter.EndObject();
```

Comment:
The structure of this object highlights the benefit in hoisting `request_id` up to the top level.

I'll probably do that before the Java side of this change, so this object structure will never exist in the Java code, but didn't want to mix two changes in this PR.